pkg/tools/glustercli-auth-header.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from datetime import datetime, timedelta
+
+import jwt
+
+
+def jwt_token(user, secret):
+    claims = dict()
+    claims['iss'] = user
+    claims['iat'] = datetime.utcnow()
+    claims['exp'] = datetime.utcnow() + timedelta(seconds=10)
+
+    token = jwt.encode(claims, secret, algorithm='HS256')
+    return (b'Authorization: bearer ' + token).decode()
+
+
+if __name__ == "__main__":
+    user = "glustercli"
+    secret_file = sys.argv[1]
+    secret = open(secret_file).read()
```

Comment:
I think its better to use:

`with open(secret_file) as f:`

instead of open().read() since file getting closed immediately is not assured. After python 3.2 it will thow a warning.ResourceWarning

Reply:
this is only development tool, errors are not handled

Reply:
I will update the tool

Reply:
Done